source/main.cc

```diff
@@ -365,6 +365,28 @@ parse_parameters (const std::string &input_as_string,
 
 
 /**
+ * Print help text
+ */
+void print_help()
+{
+  std::cout << "Usage: ./aspect [args] <parameter_file.prm>   (to read from an input file)"
+            << std::endl
+            << "    or ./aspect [args] --                     (to read from stdin)"
+            << std::endl
+            << std::endl;
+  std::cout << "    optional arguments [args]:"
+            << std::endl
+            << "       --version              (for information about library versions)"
+            << std::endl
+            << "       --help                 (for this usage help)"
+            << std::endl
+            << "       --print-xml-schema     (print xml parameters to standard output and exit)"
```

Comment:
Hm, schema is defined as follows:
```
A database schema (/ˈski.mə/ skee-mə) of a database system is its structure described in a formal language supported by the database management system (DBMS). The term "schema" refers to the organization of data...
```
In other words, it's about *how the data is structured*, not *the data itself*. In my view, XML itself is a schema.